src/audio/soundtrack.rs

```diff
@@ -22,29 +26,35 @@ impl Command for PlaySoundtrack {
     /// This command will despawn the current soundtrack, then spawn a new one
     /// if necessary.
     fn apply(self, world: &mut World) {
-        let mut soundtrack_query = world.query_filtered::<Entity, With<IsSoundtrack>>();
-        let soundtracks: Vec<_> = soundtrack_query.iter(world).collect();
-        for entity in soundtracks {
-            world.entity_mut(entity).despawn_recursive();
-        }
+        world.run_system_once_with(self, apply_play_soundstrack);
+    }
+}
 
-        let soundtrack_key = match &self {
-            PlaySoundtrack::Key(key) => key.clone(),
-            PlaySoundtrack::Disable => return,
-        };
+fn apply_play_soundstrack(
+    In(play_soundtrack): In<PlaySoundtrack>,
+    mut commands: Commands,
+    soundtracks: Query<Entity, With<IsSoundtrack>>,
```

Comment:
I know we aren't being consistent about this in other places already, but: Do we want to use the `soundtrack_query` or `soundtracks` naming convention for queries? It would be nice to make this consistent in a separate PR.

Reply:
Good point.
![image](https://github.com/user-attachments/assets/25c6bd7f-a17a-4438-9ea0-3e51601f83f6)
![image](https://github.com/user-attachments/assets/9b4a3d5e-9002-4043-9072-fa7d334b0ab8)

`_query` wins in terms of consistency. I'll change it in this PR.

Reply:
(I personally use `q_soundtrack`, but I'm not even putting that on the table as I'm sure it's too controversial)

Reply:
[`a155a1e` (#239)](https://github.com/TheBevyFlock/bevy_quickstart/pull/239/commits/a155a1ecf707a356e8c2bd47941b7f17a2a1f68c)